docs/changelog.md

```diff
@@ -0,0 +1,187 @@
+Changelog
+=========
+
+## 3.3.0 / 2020-07-19
+
+Adds a `object` backend for optimized contractions on arbitrary Python objects.
+
+**New Features**
+
+ - [\#145](https://github.com/EntosAI/codex/pull/145) Adds a `object` based backend so that `contract(backend='object')` can be used on arbitrary objects such as SymPy symbols.
```

Comment:
I think all these references to `EntosAI/codex` should be ``dgasmith/opt_einsum`` right?